front/src/reducers/osrdconf/osrdConfCommon/index.ts

```diff
@@ -64,6 +68,12 @@ export function buildCommonConfReducers<S extends OsrdConfState>(): CommonConfRe
 
 export type ConfSlice = StdcmConfSlice | OperationalStudiesConfSlice;
 
-export type ConfSliceActions = StdcmConfSliceActions | OperationalStudiesConfSliceActions;
+export type ConfSliceActions =
+  | (StdcmConfSliceActions & {
+      updateMapSettings: (payload: Partial<MapSettings>) => PayloadAction<Partial<MapSettings>>;
+    })
+  | (OperationalStudiesConfSliceActions & {
+      updateMapSettings: (payload: Partial<MapSettings>) => PayloadAction<Partial<MapSettings>>;
+    });
```

Comment:
You don't need to add manually `updateMapSettings` ;)
`StdcmConfSliceActions` is defined like this (which means takes all the actions of the slice)
```
type StdcmConfSliceActions = typeof stdcmConfSlice.actions;
```
and when you define the slice, you use `buildMapStateReducer<OsrdStdcmConfState>()`, which adds the action `updateMapSettings` to the the slice. So `updateMapSettings` is already in `stdcmConfSlice.actions`


```suggestion
export type ConfSliceActions = StdcmConfSliceActions | OperationalStudiesConfSliceActions;
```

Reply:
Thanks for the explanation, missed it ^^